content/en/security/application_security/enabling/single_step/code_security.md

```diff
@@ -208,6 +270,26 @@ Run the following commands and restart the service to stop injecting the library
 3. To disable ASM for code security, remove the `DD_IAST_ENABLED=true` environment variable from your application configuration, and restart your service.
 {{% /tab %}}
 
+{{% tab "Kubernetes" %}}
+
+1. Set the `admission.datadoghq.com/enabled:` label to `"false"` for the pod spec:
+   ```yaml
+   spec:
+     template:
+       metadata:
+         labels:
+           admission.datadoghq.com/enabled: "false"
+   ```
+2. Apply the configuration:
+   ```bash
+   kubectl apply -f /path/to/your/deployment.yaml
+   ```
+3. Restart the services you want to remove instrumentation for.
```

Comment:
```suggestion
1. Set the `admission.datadoghq.com/enabled:` label to `"false"` for the pod spec:
  
   ```yaml
   spec:
     template:
       metadata:
         labels:
           admission.datadoghq.com/enabled: "false"
   ```
2. Apply the configuration:
  
   ```bash
   kubectl apply -f /path/to/your/deployment.yaml
   ```
3. Restart the services you want to remove instrumentation for.
```